[BUG] Content installer doesn't work with datapacks

When I open up the datapacks tab in papermc this just happens

<img width="948" height="303" alt="Image" src="https://github.com/user-attachments/assets/a914d665-870e-49bd-82a6-4cf1b3b46167" />

```
/api/client/servers/8ec08dc0-9bff-42cb-bddc-f980a9bb8201/files/contents?file=%2F.mcvc-type.json:1  Failed to load resource: the server responded with a status of 404 ()
/api/client/servers/8ec08dc0-9bff-42cb-bddc-f980a9bb8201/files/contents?file=%2Fversion.json:1  Failed to load resource: the server responded with a status of 404 ()
react.C_w3zKF4.js:77 Error: Minified React error #185; visit https://react.dev/errors/185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at ai (react.C_w3zKF4.js:77:27486)
    at ni (react.C_w3zKF4.js:77:27011)
    at Is (react.C_w3zKF4.js:77:58649)
    at Fs (react.C_w3zKF4.js:77:58270)
    at I (index.20okoVDk.js:2:34609)
    at ref (index.20okoVDk.js:2:35266)
    at Xc (react.C_w3zKF4.js:77:92934)
    at Sl (react.C_w3zKF4.js:77:102368)
    at bl (react.C_w3zKF4.js:77:100026)
    at Sl (react.C_w3zKF4.js:77:100148)
Zs @ react.C_w3zKF4.js:77
ec @ react.C_w3zKF4.js:77
o.componentDidCatch.e.callback @ react.C_w3zKF4.js:77
Xa @ react.C_w3zKF4.js:77
Za @ react.C_w3zKF4.js:77
qc @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Tl @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Tl @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Ku @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
gd @ react.C_w3zKF4.js:77
dd @ react.C_w3zKF4.js:77
qu @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
hd @ react.C_w3zKF4.js:77
D @ react.C_w3zKF4.js:70
postMessage
O @ react.C_w3zKF4.js:70
e.unstable_scheduleCallback @ react.C_w3zKF4.js:70
rd @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
gd @ react.C_w3zKF4.js:77
dd @ react.C_w3zKF4.js:77
Du @ react.C_w3zKF4.js:77
ln @ react.C_w3zKF4.js:77
jd @ react.C_w3zKF4.js:77
dp @ react.C_w3zKF4.js:78
lp @ react.C_w3zKF4.js:78
index.20okoVDk.js:60 ErrorBoundary caught an error: Error: Minified React error #185; visit https://react.dev/errors/185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at ai (react.C_w3zKF4.js:77:27486)
    at ni (react.C_w3zKF4.js:77:27011)
    at Is (react.C_w3zKF4.js:77:58649)
    at Fs (react.C_w3zKF4.js:77:58270)
    at I (index.20okoVDk.js:2:34609)
    at ref (index.20okoVDk.js:2:35266)
    at Xc (react.C_w3zKF4.js:77:92934)
    at Sl (react.C_w3zKF4.js:77:102368)
    at bl (react.C_w3zKF4.js:77:100026)
    at Sl (react.C_w3zKF4.js:77:100148) {componentStack: '\n    at label (<anonymous>)\n    at Rt (https://cal…pus.strich.dev/assets/index.20okoVDk.js:60:38437)'}
componentDidCatch @ index.20okoVDk.js:60
o.componentDidCatch.e.callback @ react.C_w3zKF4.js:77
Xa @ react.C_w3zKF4.js:77
Za @ react.C_w3zKF4.js:77
qc @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Tl @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Tl @ react.C_w3zKF4.js:77
ul @ react.C_w3zKF4.js:77
Ku @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
gd @ react.C_w3zKF4.js:77
dd @ react.C_w3zKF4.js:77
qu @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
hd @ react.C_w3zKF4.js:77
D @ react.C_w3zKF4.js:70
postMessage
O @ react.C_w3zKF4.js:70
e.unstable_scheduleCallback @ react.C_w3zKF4.js:70
rd @ react.C_w3zKF4.js:77
Wu @ react.C_w3zKF4.js:77
wu @ react.C_w3zKF4.js:77
Cu @ react.C_w3zKF4.js:77
gd @ react.C_w3zKF4.js:77
dd @ react.C_w3zKF4.js:77
Du @ react.C_w3zKF4.js:77
ln @ react.C_w3zKF4.js:77
jd @ react.C_w3zKF4.js:77
dp @ react.C_w3zKF4.js:78
lp @ react.C_w3zKF4.js:78

```

--- a/frontend/src/ContentInstallerPage.tsx
+++ b/frontend/src/ContentInstallerPage.tsx
@@ -52,6 +52,9 @@ export default function ContentInstallerPage() {
       setAvailableTabs(tabs);
       setContentTab(tabs[0]);
       setSelectedWorld(result.worldDir);
+      const willShowModpacks = result.platform === 'mods' || result.platform === 'both'
+        || result.platform === 'unknown' || result.platform === 'vanilla';
+      if (!willShowModpacks) setMainTab('browse');
     }).finally(() => setDetecting(false));
   }, [server.uuid]);
 

--- a/frontend/src/ManageTab.tsx
+++ b/frontend/src/ManageTab.tsx
@@ -239,7 +239,7 @@ export default function ManageTab({ detection, contentType, installDir, refreshK
         <div className='ci-center'><Loader color='violet' size='lg' /></div>
       ) : items.length === 0 ? (
         <Text c='dimmed' ta='center' mt='xl'>
-          No {contentType === 'plugins' ? 'plugins' : 'mods'} installed.
+          No {contentType} installed.
           Browse and install some from the Browse tab!
         </Text>
       ) : (

